pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["flit","numpy==1.24.1","networkx==3.0"]
-build-backend = "flit.buildapi"
+requires = ["flit_core >=2,<4","numpy==1.24.1","networkx==3.0"]
```

Comment:
I've left the other requirements (numpy, networkx) alone, but you probably don't need them to build the package.